mypy.ini

```diff
@@ -0,0 +1,29 @@
+[mypy]
```

Comment:
Can you add comments to this file to explain why some configurations are used.

Reply:
Added the comments.